src/main.cpp

```diff
@@ -177,6 +180,8 @@ static void info(BPFnofeature no_feature)
   struct utsname utsname;
   uname(&utsname);
 
+  auto btf = bpftrace::BTF(nullptr);
```

Comment:
Nit: I’d love for the backref to get burned down, but if this is valid please just add a default constructor. Looking at this line, I have no idea what nullptr is, and it takes a lot of context chasing for the reader.